examples/006_filter_composite_data_example.py

```diff
@@ -55,16 +58,52 @@
 stress_operator.inputs.bool_rotate_to_global(False)
 stress_field = stress_operator.get_output(pin=0, output_type=dpf.types.fields_container)[0]
 
+# %%
+# Filter data by analysis ply
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+# %%
+# List all available analysis plies.
+all_ply_names = get_all_analysis_ply_names(composite_model.get_mesh())
+all_ply_names
+
+# %%
+# The easiest way to filter data by analysis ply is to use the function :func:`.get_ply_wise_data`.
+# This function supports different reduction strategies such as computing the average
+# or maximum/minimum over the spot locations.
```

Comment:
```suggestion
# This function supports different reduction strategies such as computing the average,
# maximum, or minimum over the spot locations.
```
We should avoid using ``/" as the reader doesn't know whether it means "and" or "or".